odl/phantom/transmission.py

```diff
@@ -65,12 +71,14 @@ def _modified_shepp_logan_ellipses(ellipses):
     """
     intensities = [1.0, -0.8, -0.2, -0.2, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
 
+    assert len(ellipses) == len(intensities)
```

Comment:
Well this function should not be called by any external user, ever. The only calls come through from `shepp_logan_ellipses`, and this check is to make sure that the number of ellipses in those are OK, which they currently are.

The only way to make this assert fail is by either calling a private method or by editing the ODL source code. To me that is a typical case for asserts?